docs/Readme.md

```diff
@@ -21,7 +21,6 @@
 * [Making a New Learning Environment](Learning-Environment-Create-New.md)
 * [Designing a Learning Environment](Learning-Environment-Design.md)
   * [Agents](Learning-Environment-Design-Agents.md)
```

Comment:
Kind of feels weird to have just one element indented. I would say rename `Agents` and align it?